#91857 Update plugin configuration management to contain ID (#511)
valtimo-platform/valtimo-documentation: release-notes/major11/11.2.0/valtimo-frontend-libraries.md

  The document definition displayed on the admin case detail page is now displayed using the Monaco editor in read-only
  mode.

* **Edit plugin configuration ID**

>   The plugin configuration ID can now be edited from within the plugin configuration management menu.

Reviewer: ```suggestion
  The plugin configuration ID can now be edited in the plugin configuration management screens.
```

Reviewer: Bit of a nitpicky comment, but the id can't be edited in the menu itself.
